apacheconfig/wloader.py

```diff
@@ -439,14 +496,15 @@ def __init__(self, raw, parser):
                 "``apacheconfig.parser``. First element of data is not "
                 "\"block\" typestring.")
         start = 1
-        if isinstance(raw[start], six.text_type) and raw[start].isspace():
+        if isinstance(raw[start], six.string_types) and raw[start].isspace():
```

Comment:
Changing these to `string_types` to make it more permissible; all we want to determine here is whether the text is whitespace or not, regardless of encoding.